### DISCUSSION TITLE
https://seul1230.github.io/coding/sseul-boj-16236-baby-shark

### DISCUSSION THREAD

**Discussion by giscus** [seul1230/giscus#10] [Comments]
# https://seul1230.github.io/coding/sseul-boj-16236-baby-shark

sseul's blog

https://seul1230.github.io/coding/sseul-boj-16236-baby-shark

**Comment by Cheonwooo** [comment]
설명이 잘 되있어서 이해가 잘되는 코드네요! 
포스팅 잘 보고있습니다!
